test/helper.js

```diff
@@ -7,10 +7,19 @@ const path = require('path');
 const models = require('../models');
 
 const loadFixtures = async (files) => {
-  const filePaths = files.map((f) =>
-    path.resolve(__dirname, `fixtures/${f}.json`)
-  );
-  await fixtures.loadFiles(filePaths, models);
+  const filePaths = files.map((f) => path.resolve(__dirname, `fixtures/${f}.json`));
+  await models.sequelize.transaction(async (transaction) => {
+    /// special case handling: create the first org and superuser manually, due to the circular references
+    // const org = await models.Organization.create({
+    //   id: 'f19c4a0d-227c-4929-a223-5da89f7c1d52',
+    //   name: 'Code for San Francisco',
+    //   type: 'C4SF',
+    // });
+    // const superuser = await models.User.create({
+    //   id: 'df08937c-3db7-45f6-906c-d5226c56f4f6'
+    // });
```

Comment:
Do we need to uncomment this the first time we run the tests?

Reply:
No, good catch, I'll remove this